english/bob-barker/barker.md

```diff
@@ -0,0 +1,15 @@
+The door thuds open and the light coming from within the room in front of you blinds you momentarily.  When your vision clears, standing before you is Bob Barker in all his glory.  
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.